aredis/client.py

```diff
@@ -282,7 +282,7 @@ def from_url(cls, url, db=None, skip_full_coverage_check=False, **kwargs):
         of conflicting arguments, querystring arguments always win.
         """
         connection_pool = ClusterConnectionPool.from_url(url, db=db, **kwargs)
```

Comment:
For the reason before, may it be better if `skip_full_coverage_check` is passed to `from_url`